pkg/cmd/roachtest/test_runner.go

```diff
@@ -1158,7 +1158,7 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
 	// (see runVersionUpgrade). The same is true for adding a new key to this
```

Comment:
Does this comment need to be followed to update the fixtures (as was done [for these releases](https://github.com/cockroachdb/cockroach/pull/50770))?

Reply:
Yeah, I think so. Seems like checkpointing the 20.1.5 fixtures requires the binaries to be downloadable from the homepage -- so I'm gonna wait on this PR until that's done.